### DISCUSSION TITLE
OCR

### DISCUSSION THREAD

**Discussion by Pois0nxx** [emgucv/emgucv#586] [General]
I have a error 

![image](https://user-images.githubusercontent.com/91498682/134974794-942383cd-be25-429b-8196-d4c9302f7e8b.png)

**Comment by emgucv** [comment]
Looks like you are using Emgu CV v2.4.0, it was released in 2012 and is no longer being maintained.

**Comment by Pois0nxx** [comment]
Can I run it in a new version?

27 Eyl 2021 Pzt, saat 23:22 tarihinde Emgu ***@***.***> şunu
yazdı:

**Comment by emgucv** [comment]
Sure, if you are use nuget pacake, you can get it from here:
https://www.nuget.org/packages/Emgu.CV.runtime.windows/

Some code changes may be needed moving from 2.4.0 to the current 4.5.3 release.
